index.html

```diff
@@ -67,7 +67,7 @@ <h2>Projects</h2>
         
         
           <div class="project-container project-1">
-            <img src="images/project-1.jpg" alt="project-1 image">
+            <img src="images/project-1.jpg" alt="Hands typing on a macbook pro 13'', a notebook, an orange, a pen and a smartphone">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.